packages/button/src/react/__specs__/index.spec.tsx

```diff
@@ -1,80 +1,107 @@
-import { fireEvent, render } from '@testing-library/react'
+import { PlaceholderIcon } from '@pluralsight/ps-design-system-icon'
+import { convertStoriesToJestCases } from '@pluralsight/ps-design-system-util'
+import { fireEvent, screen, render } from '@testing-library/react'
+import { axe } from 'jest-axe'
 import React from 'react'
 
-import { CheckIcon } from '@pluralsight/ps-design-system-icon'
-
 import Button from '..'
+import * as vars from '../../vars'
+
+import * as stories from '../__stories__/index.story'
 
 describe('Button', () => {
-  it('should render a button', () => {
-    const { getByTestId } = render(
-      <Button data-testid="undertest" type="submit">
-        test
-      </Button>
-    )
-    const el = getByTestId('undertest')
-
-    expect(el.tagName.toLowerCase()).toEqual('button')
-  })
+  const cases = convertStoriesToJestCases(stories)
 
-  it('should render a hyperlink when an `href` is present', () => {
-    const { getByTestId } = render(
-      <Button data-testid="undertest" href="/">
-        test
-      </Button>
-    )
-    const el = getByTestId('undertest')
+  describe('.sizes', () => {
+    it('exists', () => {
+      expect(Button.sizes).toEqual(vars.sizes)
+    })
+  })
 
-    expect(el.tagName.toLowerCase()).toEqual('a')
+  it('should forward ref', () => {
```

Comment:
```suggestion
  it('forwards ref', () => {
```

If you're interested, while we're working through each of these specs, I fully welcome dropping the repeating "should" from descriptions.

Reply:
yeah now it a great time. using should is just the convention i've typically followed - but happy to adopt a new standard. if you can summarize the approach i'll add it to the issue. maybe `use concise test names`?

Reply:
Sure, I can add it to our style guide.  

Section 1.1.b
Be it known that the spec description shall not include the word "should". This is right.